src/Http/Controllers/Controller.php

```diff
@@ -53,9 +53,11 @@ public function insertUpdateData($request, $slug, $rows, $data)
             $content = $this->getContentBasedOnType($request, $slug, $row);
 
             // Set the content back to the previous value when there is really now input for this field
-            if (is_null($content) && is_null($request->input($row->field))) {
-                if (isset($data->{$row->field})) {
-                    $content = $data->{$row->field};
+            if (is_null($content)) {
+                if (is_null($request->input($row->field))) {
+                    if (isset($data->{$row->field})) {
```

Comment:
This can be combined into the `if` above it.  I'm not a huge fan of cascading conditionals, especially when they're related like this.